src/utils/CiceroMarkToOOXML/index.js

```diff
@@ -96,6 +125,18 @@ const renderNodes = (context, node, counter, parent=null) => {
     insertSoftBreak(context);
     return;
   }
+  if (node.$class === definedNodes.listBlock || node.$class === definedNodes.list) {
```

Comment:
@DianaLease as you can see here, they are both rendered the same way. The only reason was we weren't able to render variables inside the list. I hope the solution we found works :crossed_fingers: